error compiling on fedora

```bash
$cmake ..
all ok
$make
...
[ 85%] Linking CXX executable unit_rtp_test
[ 85%] Built target unit_rtp_test
[ 85%] Building CXX object CMakeFiles/unit_DHT_test.dir/toxcore/DHT_test.cc.o
/home/oooo/c-toxcore/toxcore/DHT_test.cc:3:10: fatal error: gmock/gmock.h: Nie ma takiego pliku ani katalogu
    3 | #include <gmock/gmock.h>
      |          ^~~~~~~~~~~~~~~
compilation terminated.
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -146,6 +146,8 @@ endif()
 option(BUILD_MISC_TESTS "Build additional tests" OFF)
 option(BUILD_FUN_UTILS "Build additional just for fun utilities" OFF)
 
+option(UNITTEST "Enable unit tests (disable if you don't have a working gmock or gtest)" ON)
+
 option(AUTOTEST "Enable autotests (mainly for CI)" OFF)
 if(AUTOTEST)
   option(NON_HERMETIC_TESTS "Whether to build and run tests that depend on an internet connection" OFF)
@@ -534,7 +536,7 @@ endfunction()
 
 # The actual unit tests follow.
 #
-if(TARGET GTest::gtest AND TARGET GTest::gmock)
+if(UNITTEST AND TARGET GTest::gtest AND TARGET GTest::gmock)
   unit_test(toxav ring_buffer)
   unit_test(toxav rtp)
   unit_test(toxcore DHT)

--- a/INSTALL.md
+++ b/INSTALL.md
@@ -151,6 +151,7 @@ There are some options that are available to configure the build.
 | `PROXY_TEST`            | Enable proxy test (requires `other/proxy/proxy_server.go` to be running).                     | ON or OFF                                                                 | OFF                                               |
 | `STRICT_ABI`            | Enforce strict ABI export in dynamic libraries.                                               | ON or OFF                                                                 | OFF                                               |
 | `TEST_TIMEOUT_SECONDS`  | Limit runtime of each test to the number of seconds specified.                                | Positive number or nothing (empty string).                                | Empty string.                                     |
+| `UNITTEST`              | Enable unit tests (disable if you don't have a working gmock or gtest).                       | ON or OFF                                                                 | ON                                                |
 | `USE_IPV6`              | Use IPv6 in tests.                                                                            | ON or OFF                                                                 | ON                                                |
 
 You can get this list of option using the following commands

